index.html

```diff
@@ -17,28 +17,31 @@
         <div id="main">
             <div id="left">
                 <img src="pics/openageinverse.svg" alt="openage" id="logo">
-                <h1>A free (as in <i>freedom</i>) clone of the Age of Empires II engine</h1>
+                <h1>Free cross-platform RTS game engine based on Age of Empires mechanics</h1>
```

Comment:
I would rather like the phrase "[...] RTS engine providing Age-of-Empires-style game mechanics" because that makes it a bit clearer what openage offers.

Reply:
Yeah, sounds better. Also "providing" sound much more ** actively supportive** then a passive "based" on.